packages/mdc-chips/chip-set/foundation.ts

```diff
@@ -69,19 +69,11 @@ export class MDCChipSetFoundation extends MDCFoundation<MDCChipSetAdapter> {
    * Selects the chip with the given id. Deselects all other chips if the chip set is of the choice variant.
    */
   select(chipId: string) {
```

Comment:
Since this is not used anywhere outside foundation can we remove this select and rename selectOnInit to select? it makes it ambiguous to have separate select APIs.

Reply:
Done